### PR TITLE
fix(quantic): SmartSnippet to open in a new tab

### PR DESCRIPTION
**Pull request by tedre191**
Fix to the links in Quantic SmartSnippet to open in a new tab, instead of the current tab.
https://coveord.atlassian.net/browse/SFINT-5451

### CODE REVIEW

**Comment by mmitiche** [comment]
Fix look good 👍

## Demo:

#### Before

https://github.com/coveo/ui-kit/assets/86681870/a59515e9-f633-4fba-b9dc-4d9144c83d10

#### After


https://github.com/coveo/ui-kit/assets/86681870/bfe71813-69c1-4d44-a41c-51b627e1a937

**Review by mmitiche — APPROVED** [review]
LGTM!

File: packages/quantic/force-app/main/default/lwc/quanticSmartSnippetAnswer/quanticSmartSnippetAnswer.js [hunk 1]
```diff
@@ -69,6 +69,8 @@ export default class QuanticSmartSnippetAnswer extends LightningElement {
     const disabledCSSClass = 'inline-link--disabled';
     this.inlineLinks.forEach((link) => {
       if (link?.href) {
+        link.target = '_blank';
```
Comment (tedre191) [inline packages/quantic/force-app/main/default/lwc/quanticSmartSnippetAnswer/quanticSmartSnippetAnswer.js:72]: @MehdiMitiche should we put this outside of the "if" ? Not sure what use case we have where the href is null.

**Comment by mmitiche** [comment]
Fixed issue where going the next page was causing issues with the smart snippet sources.

https://github.com/coveo/ui-kit/assets/86681870/7e249380-8b88-4cc1-874b-5ed1c0c43c90

**Pull request merged**